### ISSUE TITLE
Custom iPod Music?

### ISSUE THREAD

**Issue by Bommel24** [GHzGangster/Solid4#1]
Would it be possible to create custom Ipod Music with this?

**Comment by GHzGangster** [comment]
Yes, you'd want to load an iPod song, re-encode with your new audio, and export it. Should work fine, we do the same for MGO. This code is really old though, and I've made a GUI since, but I never finished it.
